example_project/example/models/example/http_events_bytes_sent.sql

```diff
@@ -1,4 +1,4 @@
-{{ config(primary_key="method") }}
+{{ config(primary_keys=["method"]) }}
 
 SELECT CAST(envoy['method'] AS STRING)       AS `method`,
```

Comment:
I stole this from you. It gets this example working again for me:

```suggestion
SELECT coalesce(CAST(envoy['method'] AS STRING), '__UNKNOWN__')  AS `method`,
```

Before I was getting this error:

> Invalid stream. Reason: Invalid schema: Invalid primary key 'PK_method'. Column 'method' is nullable.

In combination with the watermark tweak I mentioned, this enabled me to run the full example without `--select`-ing a specific resource 🙌